mifospay/src/main/java/org/mifos/mobilewallet/mifospay/registration/ui/SignupActivity.java

```diff
@@ -246,6 +247,12 @@ public void onNextClicked() {
             return;
         }
 
+        if (!Utils.isValidEmail(email)) {
+            Toaster.showToast(this, "Email not valid");
+            hideProgressDialog();
+            return;
+        }
+
```

Comment:
This extra line is added to match with code structure already used in file. Also, this makes code appear a bit clean and more readable.